Description: Make sphinx-build ignore .pc directory
 The quilt patch management tool might create the '.pc' directory during
 the package build process. In case that '.rst' files are patched by
 quilt, they end up in '.pc' as well. A sphinxdoc recusively scans the
 sources for '.rst' files, this causes unwanted behaviour. This patch
 adds '.pc' to the list of excluded patterns.
Forwarded: https://gitlab.com/mailman/mailman/merge_requests/320
Author: Jonas Meurer <jonas@freesources.org>
Last-Update: 2017-09-20
From: Jonas Meurer <jonas@freesources.org>
Date: Wed, 20 Sep 2017 16:15:39 +0200
Subject: _sphinx_ignore_pc

---
 conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/conf.py b/conf.py
index 46790c1..8d217c1 100644
--- a/conf.py
+++ b/conf.py
@@ -68,7 +68,7 @@ release = VERSION
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = ['_build', 'eggs', '.tox']
+exclude_patterns = ['_build', 'eggs', '.tox', '.pc']
 
 # The reST default role (used for this markup: `text`) to use for all documents.
 #default_role = None
